CMakeLists.txt

```
@@ -86,12 +86,19 @@ else()
   option(DISABLE_DEPRECATION_WARNINGS "Disable depreaction warnings " ON)
   option(NVTX "Enable nvtx markers" OFF)
   set(BACKEND_FOLDER "/opt/tritonserver/backends" CACHE STRING "Triton backend folder path")
+  set(RAPIDS_DEPENDENCIES_VERSION "22.04" CACHE STRING "RAPIDS projects dependencies version")
+  option(TRITON_FIL_USE_TREELITE_STATIC "Link Treelite statically in libtriton_fil.so and cuml++.so" ON)
+  option(TRITON_FIL_ENABLE_TREESHAP "Compile GPU Treeshap when building cuml++.so" ON)
+
 
   message(VERBOSE "RAPIDS_TRITON_BACKEND: Enabling detection of conda environment for dependencies: ${DETECT_CONDA_ENV}")
   message(VERBOSE "RAPIDS_TRITON_BACKEND: Enabling kernelinfo in nvcc: ${CUDA_ENABLE_KERNEL_INFO}")
   message(VERBOSE "RAPIDS_TRITON_BACKEND: Enabling lineinfo in nvcc: ${CUDA_ENABLE_LINE_INFO}")
   message(VERBOSE "RAPIDS_TRITON_BACKEND: Enabling nvtx markers: ${NVTX}")
   message(VERBOSE "RAPIDS_TRITON_BACKEND: Build RAPIDS_TRITON_BACKEND unit-tests: ${BUILD_TESTS}")
+  message(VERBOSE "RAPIDS_TRITON_BACKEND: RAPIDS projects dependencies version: ${RAPIDS_DEPENDENCIES_VERSION}")
+  message(VERBOSE "RAPIDS_TRITON_BACKEND: Linking Treelite statically: ${FIL_TRITON_USE_TREELITE_STATIC}")
+  message(VERBOSE "RAPIDS_TRITON_BACKEND: Compiling GPU Treeshap in cuml++.so ${TRITON_FIL_ENABLE_TREESHAP}")
```

```suggestion
  message(VERBOSE "RAPIDS_TRITON_BACKEND: Compiling GPU Treeshap in libcuml++.so ${TRITON_FIL_ENABLE_TREESHAP}")
```

Verbosity to help newcomers